capa/features/extractors/ida/insn.py

```diff
@@ -213,7 +213,11 @@ def extract_insn_offset_features(
             continue
 
         p_info = capa.features.extractors.ida.helpers.get_op_phrase_info(op)
-        op_off = p_info.get("offset", 0)
+
+        op_off = p_info.get("offset", None)
+        if op_off is None:
+            continue
+
```

Comment:
do not assume `0` if `offset` is not present; ideally we'd create a class for phrase info instead of using a dictionary but I do not think it's worth the effort right now.